boards/arm/sama5/giant-board/src/sam_sdmmc.c

```diff
@@ -0,0 +1,319 @@
+/****************************************************************************
+ *  boards/arm/sama5/giant-board/src/sam_sdmmc.c
+ *
```

Comment:
This entire file must be removed.  The Giant Board does not support the SDMMC (HSMCI) interface.  The interface to the SD card has to be via SPI.